london-emotions/lib.py

```diff
@@ -56,6 +56,6 @@ def clean_data(data):
     # For introspections purpose to quickly get this functions on ipython
     import london-emotions
     folder_source, _ = split(london-emotions.__file__)
-    df = pd.read_csv('{}/data/data.csv.gz'.format(folder_source))
+    df = pd.read_csv('{}/data/data_train.csv'.format(folder_source))
```

Comment:
Did you change the name of the csv file? 

Maybe we can add that later because I think the csv file as another name